### DISCUSSION TITLE
Extending Existing Plugins

### DISCUSSION THREAD

**Discussion by BenjaminCubbage** [Vector35/binaryninja-api#2383] [API] [answered]
I want to extend the existing Debugger plugin. Is there a simple API for accessing an installed plugin's context without having to create an entirely new plugin and copy all of the existing plugin's code into it?

**Comment by plafosse** [comment, marked as answer]
No there isn't a simple API for accessing it. There are a few ways for doing this.

1.  Hacky way is accessing it through `debugger = sys.modules['Vector35_debugger']`
2. You could directly modify the debugger plugin code yourself to suit your needs
